fxevent/zap.go

```diff
@@ -118,6 +118,7 @@ func (l *ZapLogger) LogEvent(event Event) {
 				zap.String("constructor", e.ConstructorName),
 				moduleField(e.ModuleName),
 				zap.String("type", rtype),
+				zap.Bool("private", e.Private),
```

Comment:
Consider including this only if private is true. You can do that with something like the moduleField function:

```go
func trueBool(name string, b bool) zap.Field {
  if b {
    return zap.Bool(name, true)
  }
  return zap.Skip()
}
```

This will reduce noise in logs since private provides are rare.